scripts/uosc/elements/PauseIndicator.lua

```diff
@@ -5,7 +5,7 @@ local PauseIndicator = class(Element)
 
 function PauseIndicator:new() return Class.new(self) --[[@as PauseIndicator]] end
 function PauseIndicator:init()
```

Comment:
I didn't verify this in a test, but I think this needs to call `decide()`, because otherwise the static pause indicator won't show if a new pause indicator gets created because of a script-message.

Option changes cause `on_prop_pause()` to be called, so it's not a problem there.

Reply:
I'll add it.